packages/patternfly-4/react-core/src/components/Page/Page.tsx

```diff
@@ -31,6 +31,10 @@ export interface PageProps extends React.HTMLProps<HTMLDivElement> {
    * the sidebar component or add a callback onNavToggle function into the PageHeader component
    */
   isManagedSidebar?: boolean;
+  /**
+   * If true, the managed the sidebar is initially open for desktop view
+   */
+  isManagedSidebarOpen?: boolean;
```

Comment:
It might be clearer that this sets the default behavior by naming the prop `defaultManagedSidebarOpen` instead. Otherwise it would seem that you're able to change the prop programmatically after the page is already created.